### ISSUE TITLE
/usr/local/bin/gg: line 281: [: too many arguments

### ISSUE THREAD

**Issue by n10v** [qw3rtman/gg#60]
When I use command `gg c <message>`, this warning appears: `/usr/local/bin/gg: line 281: [: too many arguments`
![2015-08-30 0 14 22](https://cloud.githubusercontent.com/assets/13235519/9563857/4ed6f760-4eac-11e5-9088-77f4e3b47991.png)

**Comment by schikulski** [comment]
Not sure if this is related, but I get this message when running `gg c`

```
gg c "fixes #2"                                                                                                               ⏎
[✔] Added everything!
/usr/local/bin/gg: line 281: [: fixes: binary operator expected
[✔] Committed!

[master|5365ad2] fixes #2
```

**Comment by n10v** [comment]
@schikulski Hm. Maybe we use different versions, therefore we get different messages but on the same line.
I use 2.0.0

**Comment by schikulski** [comment]
2.0.0 here as well

**Comment by n10v** [comment]
Someone should fix that :(

**Comment by qw3rtman** [comment]
This issue should be fixed now thanks to #59 and #63. Let me know if it is fixed.
